src/styles/app.css

```
@@ -126,6 +126,278 @@ button {
+.ts-mega-trigger-wrap {
+  display: inline-flex;
+}
+
+.ts-mega-trigger::after {
+  content: '';
+  position: absolute;
+  right: 0.625rem;
+  bottom: 0.3rem;
+  left: 0.625rem;
+  height: 1px;
+  background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)